ui/js/component/transactionList/view.jsx

```diff
@@ -23,6 +25,25 @@ class TransactionList extends React.PureComponent {
     return !filter || filter == transaction.type;
   }
 
+  isRevokeable(txid, nout) {
+    // a claim/support/update is revokable if it
+    // is in my claim list(claim_list_mine)
+    return this.props.myClaims.has(`${txid}:${nout}`);
+  }
+
+  revokeClaim(abandonData) {
+    const {
+      name: name,
+      claimId: claimId,
+      txid: txid,
+      nout: nout,
+    } = abandonData;
+
+    const uri = lbryuri.build({ name, claimId });
+    this.props.resolveUri(uri);
```

Comment:
Why does this happen here? If the modal needs it, it should probably be in the modal.

Reply:
I am getting rid of this resolve before revoking all together, and trigger it after revoking. That would eliminate the need to make the synchronous dispatch calls too.